src/v2/guide/index.md

```diff
@@ -73,6 +73,8 @@ var app = new Vue({
 
 We have already created our very first Vue app! This looks pretty similar to rendering a string template, but Vue has done a lot of work under the hood. The data and the DOM are now linked, and everything is now **reactive**. How do we know? Open your browser's JavaScript console (right now, on this page) and set `app.message` to a different value. You should see the rendered example above update accordingly.
 
+Note that we no longer have to interact with the HTML directly. A Vue app attaches itself to a single, global DOM element (`#app` in our case) then fully controls it. The HTML is our entry point, but everything else happens within the newly created Vue instance.
```

Comment:
This is great! I have but one tiny issue: What's a _global_ DOM element? Can/should we refer to it more simply as a "single DOM element" to keep beginners from getting further confused?

Reply:
Yup you're right, I wasn't too sure about it as well. Fixing this right now!